### ISSUE TITLE
No way of debugging?

### ISSUE THREAD

**Issue by erikbsap** [rook/rook#1770]
Hi, I seem to be too stupid, but the suggestion to use rook slack seems to require me to have a rook.io email address when registering. So sadly I can't join there.

So, I'm trying to debug a rook on k8s issue. After some time (usually 3 days) the whole thing dies. And it seems to kill the nodes with it. They are still listed, but state is "NotReady" and it's not possible to retreive any logs, or `df -h` on the hosts for that matter.

My question is this: How can one make sure that at least not everything dies. Let's assume for now the LVM volume for rook is full and that's why it has a problem. Can I tell rook via config "please die when you reach 95% load on your given volume"?

Also I checked all the `rook*` logs manually because thankfully docker stores the logs in files in another volume. But the only errors/warnings are the following which I assume is happening after the problems start:
 - `2018-06-05 11:14:18.925806 W | cephosd: Failed OSD status check: failed to get osd dump: exit status 1\n","stream":"stderr","time":"2018-06-05T11:14:18.926011966Z`
 - `2018-06-04 06:30:55.027182 W | op-mon: failed to find initial monitor rook-ceph-mon1 in mon map\n","stream":"stderr","time":"2018-06-04T06:30:55.027465899Z`
 - `2018-06-05 11:07:18.377192 E | op-mon: failed to failover mon rook-ceph-mon2. failed to start new mon rook-ceph-mon3. failed to wait for mon quorum. exceeded max retry count waiting for monitors to reach quorum\n","stream":"stderr","time":"2018-06-05T11:07:18.377487297Z`

Basically it's not clear to me how I can start to debug here.

**Comment by galexrt** [comment]
@erikbsap You can use whatever email you want to register for the Slack.

Please fill out the issue template for this issue as we need information about your environment, see https://raw.githubusercontent.com/rook/rook/master/.github/ISSUE_TEMPLATE.md.

**Comment by erikbsap** [comment]
I know the following doesn't sound too polite. Sorry for that. Not sure I can formulate it more polite. Please assume it's the heat in our unclimatized office if you feel it's not good enough. Thanks.

One cannot choose the part behind the @ in the slack registration. It is not part of the text field.

The rook docker image is version `v0.7.0-106.gb9ae002`, everything else *about debugging* should be the same in every environment, right? There should be some logs/events to read somewhere. There should be some bash commands to query. And finally maybe some config parameters of rook to set, e.g. log level.

**Comment by galexrt** [comment]
@erikbsap To register for the Rook.io Slack you go to https://rook.io/ and click the Slack button in the left corner, then you should get to https://rook-slackin.herokuapp.com/. There you can register with any email for the Rook.io Slack.

The environment information is important so that we can guide you in the right direction.
For general debugging, see this doc page https://rook.io/docs/rook/master/common-issues.html#troubleshooting-techniques.

**Comment by tonya11en** [comment]
Verified you can log in with any email via https://rook-slackin.herokuapp.com/.
